src/modules/commander/Commander.cpp

```diff
@@ -2118,7 +2118,7 @@ Commander::run()
 
 		} else {
 			if (!status_flags.rc_input_blocked && !status.rc_signal_lost) {
-				mavlink_log_critical(&mavlink_log_pub, "MANUAL CONTROL LOST (at t=%" PRIu64 "ms)", hrt_absolute_time() / 1000);
+				mavlink_log_critical(&mavlink_log_pub, "Manual control lost (at t=%" PRIu64 "ms)", hrt_absolute_time() / 1000);
```

Comment:
In my opinion we should remove the timestamp here.

Reply:
Fine by me. I always wondered why this message should be exceptional :sweat_smile: